test/test_scalar_hoisting.cpp

```diff
@@ -334,7 +332,7 @@ __global__ void CUDAGeneratedKernel(Tensor<float, 2, 2> T0, Tensor<float, 2, 2>
 
   assertCUDAKernel(fusion.get(), expected_kernel);
 
-  testValidate(fusion.get(), cg_outputs, {t0}, {t1}, __LINE__, __FILE__);
```

Comment:
Should the computation of `t1` above be removed?

Reply:
Done. `clangtidy` does report unused variables but does not raise an error here.